app/controllers/api/v1/merchants/revenue_controller.rb

```diff
@@ -7,7 +7,12 @@ def index
   end
 
   def show
-    revenue = Merchant.revenue_across_all_transactions(params[:id])
-    respond_with revenue
+    if params[:date]
```

Comment:
@jmejia How could we get around using an if statement in this controller seeings as how the route is the same aside from the parameter of date?